### PR TITLE
[15.0][MIG] sale_stock_picking_note

### PR DESCRIPTION
**Pull request by juanpgarza**
Standard migration

### CODE REVIEW

**Review by legalsylvain — CHANGES_REQUESTED** [review]
Could you split sale_stock.py, a file per model ?

Otherwise,lgtm. Code review / no test.

**Comment by rousseldenis** [comment]
@juanpgarza Thanks for this. Could you check tests ?

**Comment by juanpgarza** [comment]
> @juanpgarza Thanks for this. Could you check tests ?

@rousseldenis I can't see the error. Could you help me please?

**Comment by JuanyDForgeflow** [comment]
@juanpgarza The test error is not related to this module but on the sale_automatic_workflow module

image

File: sale_stock_picking_note/models/sale_order.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    picking_note = fields.Html(string="Picking Internal Note")
```
Comment (rousseldenis) [inline sale_stock_picking_note/models/sale_order.py:11]: @juanpgarza Why changing this to Html ?

**Comment by AaronHForgeFlow** [comment]
The module is migrated. This can be closed.

**Pull request closed without merging**